CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## v0.9.0-rc28 [unreleased]
 
+### Bugfixes
+- [#2374](https://github.com/influxdb/influxdb/issues/2374): Two different panics during SELECT percentile
```

Comment:
I know this is a small thing, but I am pretty sure @pauldix does not want this format. He specially told me links to PRs only a few months back. Not a big deal, but you should check with him when you get a chance.